examples/codestral-code-interpreter-js/README.md

```diff
@@ -0,0 +1,56 @@
+# AI Code Execution with Mistral's Codestral
+
+This project tests the capabilities of the new Mistral's model on data analysis tasks, using the [Code Interpreter SDK](https://github.com/e2b-dev/code-interpreter) by E2B. Codestral doesn't support using tools yet, so in this Python example, we added the code interpreting capabilities.
+
+We are going to build an AI assistant that performs data analysis on a CSV file. It is prompted to plot the average temperature over the years in Algeria,
+
+
+# Installation
+
+First, cone the E2B Cookbook repository, and navigate to the correct directory.
```

Comment:
Typo - "cone". You don't need to mention it though, people know that. First step is installing dependencies